CHANGELOG.md

```diff
@@ -1,3 +1,76 @@
+# v 1.12.0 (2022-01-24)
+
+## OS Changes
+
+* Add patch to enable strict aliasing in dbus-broker ([#2730])
```

Comment:
this could go in build changes instead, since it was related to the SDK update
```suggestion
* Disable strict aliasing for c-utf8 library in dbus-broker ([#2730])
```